### DISCUSSION TITLE
添加一下Tag支持？

### DISCUSSION THREAD

**Discussion by neokoni** [MaLuns/hexo-theme-async#95] [Ideas 奇思妙想]
hexo上写的原版tag比较单调，以前用的{% note %}这种。可否添加一下对tag之类的支持而不是额外安装插件？  
类似于这样
![20231003123159_1](https://github.com/MaLuns/hexo-theme-async/assets/90206655/ba1bde1b-be9c-4ab3-bf2e-929854cfe4bd)
可以的话希望是类似与async文档的那种风格

**Comment by MaLuns** [comment]
在下个小版本，加上这个
![image](https://github.com/MaLuns/hexo-theme-async/assets/31614024/13040472-9441-441c-95bb-77d3afc0930f)
